smarts/core/mission_planner.py

```diff
@@ -258,7 +263,6 @@ def cut_in_waypoints(self, sim, pose: Pose, vehicle):
             lane_id = lane.getID()
             lane_index = lane_id.split("_")[-1]
             width = lane.getWidth()
-            speed_limit = lane.getSpeed()
```

Comment:
Did you intend to apply the `speed_limit` variable here?

Reply:
Thanks, no, it is removed from here, I set the value above.

Reply:
My bad, I did not see the application further down.